### PR TITLE
Feat/mr 16

### PR DESCRIPTION
**Pull request by Arzoid29**

### CODE REVIEW

File: src/app/why-sales-assist/page.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+export default function page() {
+  return (
+    <main>
+      <Header></Header>
+      <WhySalesAssistHero data={dataHero}></WhySalesAssistHero>
+      <Benefits data={dataBenefits}></Benefits>
+    </main>
+  );
+}
```
Comment (brya0x) [inline src/app/why-sales-assist/page.tsx:16]: **Question**: this works i though it has to be open cased

File: src/components/atoms/insight/index.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+            <div className="flex items-center mb-4">
+                <MdInsights className="text-[1.5rem]" />
+                <h4 className="ml-3 font-bold">{text}</h4>
+            </div>
+            <div className="pl-12 md:p-0 xl:w-2/3">
+                <h5 className="text-white">{children}</h5>
+            </div>
+        </div>
+    )
+}
```
Comment (brya0x) [inline src/components/atoms/insight/index.tsx]: **Change**: remember use function over const

File: src/components/organism/sections/why-sales-assist/benefits/index.tsx [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+  }, [currentCard, progressLeft]);
+
+  function clickOnIcon(card: number, startAt: number) {
+    return () => {
+      currentCard.current = card;
+      startProgress();
+    };
+  }
+
+  const isActive = (at: number) => currentCard.current === at;
```
Comment (brya0x) [inline src/components/organism/sections/why-sales-assist/benefits/index.tsx]: **Change**: you are not using the hook approach

File: src/components/organism/sections/why-sales-assist/benefits/index.tsx [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+                <div className="w-full flex items-center  ">
+                  <IconBox
+                    text="Customer"
+                    icon="/images/SA_Customer_Icon.svg"
+                    isActive={isActive(2)}
+                    onClick={clickOnIcon(2, 373)}
+                    progress={progressLeft}
+                  />
+                </div>
+              )}
```
Comment (brya0x) [inline src/components/organism/sections/why-sales-assist/benefits/index.tsx]: **Change**: mmhh what the fuck this is to complex, refactor it

File: src/model/interactions/use-window-size/useWindowSize.ts [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+      handleResize()
+    
+      return () => {
+        window.removeEventListener('resize', handleResize)
+      };
+    }, []);
+
+
+    return { ...size }
+}
```
Comment (brya0x) [inline src/model/interactions/use-window-size/useWindowSize.ts:35]: **Change**: everything is okay but why is call useResponsive the file use index.tsx is easier

**Review by code-bryan — CHANGES_REQUESTED** [review]

File: src/components/atoms/icon-box/index.tsx [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+                        <circle
+                            className="percent fifty stroke-orange-500 "
+                            cx="80" cy="80" r="72"
+                            fill="none"
+                            strokeWidth="4"
+                            pathLength="100"
+                            strokeDashoffset={progress || 100}
+                            strokeDasharray={100}
+                        />
+                    </svg>}
```
Comment (brya0x) [inline src/components/atoms/icon-box/index.tsx]: **Change**: move svgs to a directory under atoms call icons instead of using those like this

**Review by code-bryan — CHANGES_REQUESTED** [review]

**Review by code-bryan — APPROVED** [review]

**Pull request merged**